src/main/java/com/github/manolo8/darkbot/gui/MainGui.java

```diff
@@ -33,8 +39,22 @@ public MainGui(Main main) throws HeadlessException {
         ToolTipManager.sharedInstance().setInitialDelay(350);
 
         setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
-        setSize(640, 480);
-        setLocationRelativeTo(null);
+
+        Config.BotSettings botSettings = main.config.BOT_SETTINGS;
+        Runtime.getRuntime().addShutdownHook(new Thread(() -> {
+            if (botSettings.SAVE_GUI_POS)
+                main.configManager.saveConfig();
```

Comment:
```suggestion
            main.configManager.saveConfig();
```
I think always saving config isn't a bad idea